### DISCUSSION TITLE
How to import ParametricExpressionSpec for Parametric Expression?

### DISCUSSION THREAD

**Discussion by saremnorouzi** [MilesCranmer/PySR#754] [General]
Hi,

I saw in the release video for parametric expressions that Miles writes the following:

expression_spec=ParametricExpressionSpec(max_parameters=2)

However, when I try a similar exercise in Python, I am unable to import ParametricExpressionSpec properly.

I would appreciate it if someone could clarify this for me.

Best regards,
Sarem

**Comment by MilesCranmer** [comment]
It just got released today, try updating?

**Comment by saremnorouzi** [comment]
I had updated but update it again and now works fine :)
